examples/alert.json

```diff
@@ -2,23 +2,29 @@
 	"apiVersion": "synopsys.com/v1",
 	"kind": "Alert",
 	"metadata": {
-	  "clusterName": "",
-	  "name": "alert-test"
+		"clusterName": "",
+		"name": "alert-test"
 	},
 	"spec": {
-	  "namespace": "alert-test",
-	  "registry": "docker.io",
-	  "imagePath": "blackducksoftware",
-	  "alertImageName": "blackduck-alert",
-	  "alertImageVersion": "2.1.0",
-	  "cfsslImageName": "hub-cfssl",
-	  "cfsslImageVersion": "4.8.1",
-	  "blackduckHost": "<<HUB_HOST>>",
-	  "blackduckUser": "sysadmin",
-	  "blackduckPort": 443,
-	  "port": 8443,
-	  "standAlone": true,
-	  "alertMemory": "512M",
-	  "cfsslMemory": "640M"
+		"namespace": "alert-test",
+		"registry": "docker.io",
+		"imagePath": "blackducksoftware",
+		"alertImageName": "blackduck-alert",
+		"alertImageVersion": "3.1.0",
+		"cfsslImageName": "blackduck-cfssl",
+		"cfsslImageVersion": "1.0.0",
+		"port": 8443,
+		"standAlone": true,
+		"alertMemory": "2560M",
+		"cfsslMemory": "640M",
+		"pvcMemory": "100M",
+		"pvcStorageClass": "standard",
+		"environs": [
+			"ALERT_SERVER_PORT:8443",
+			"PUBLIC_HUB_WEBSERVER_HOST:<<HOST>>",
+			"PUBLIC_HUB_WEBSERVER_PORT:443",
+			"ALERT_ENCRYPTION_PASSWORD:<<PASSWORD>>",
+			"ALERT_ENCRYPTION_GLOBAL_SALT:<<GLOBALSALT>>"
```

Comment:
Could ALERT_ENCRYPTION_PASSWORD and ALERT_ENCRYPTION_GLOBAL_SALT be automatically generated by the operator?

Reply:
Paulo said that the preferred way is to have the user enter them. However, it could be useful if synopsysctl had a flag that would autogenerate them for the customer.

Reply:
I can add code that checks the number of characters if they are entered.

Reply:
I added code to check these flags in synopsysctl and when the Secret is being created with the values.